roles/rsyslog/tasks/main.yml

```diff
@@ -77,20 +84,23 @@
         path: ["{{ __rsyslog_config_dir }}", /etc/rsyslog.conf]
         dest: "{{ rsyslog_backup_dir }}/backup.tgz"
         remove: '{{ true if __rsyslog_purge_original_conf | bool else false }}'
+        mode: '0600'
       changed_when: false
 
-    - name: Create logging directory if it does not exist or the ownership and/or modes are different.
+    - name: "Create logging directory if it does not exist or
+      the ownership and/or modes are different."
       file:
         state: directory
         path: '{{ __rsyslog_system_log_dir }}'
         owner: 'root'
         group: 'root'
-        mode: '0700'
+        mode: '0755'
 
     - name: Generate main rsyslog configuration
       template:
         src: 'rsyslog.conf.j2'
         dest: '/etc/rsyslog.conf'
+        mode: preserve
```

Comment:
Note that `preserve` doesn't mean "preserve the existing mode on the existing file" - it means "the mode on the `dest` file should be exactly the same as the mode on the `src` file" which is probably not what you intended - see https://docs.ansible.com/ansible/2.8/modules/copy_module.html#copy-module
"`preserve` means that the file will be given the same permissions as the source file."

You could do this by omitting the `mode`, but then ansible-lint will complain.

I think the only way to keep the existing mode, while at the same time avoiding the ansible-lint warning, is to use the `stat` module to get the file mode, something like this:
```yaml
- name: Get mode of rsyslog.conf if it exists
  stat:
    path: /etc/rsyslog.conf
  register: __rsyslog_register_stat_conf

- name: Generate main rsyslog configuration
  template:
    src: 'rsyslog.conf.j2'
    dest: '/etc/rsyslog.conf'
    mode: "{{ __rsyslog_register_stat_conf.stat.mode | d('0700') }}"
```